plugin/src/test/kotlin/com/github/weisj/darkmode/platform/linux/gnome/GnomeNativeTest.kt

```diff
@@ -76,7 +78,7 @@ class GnomeNativeTest {
         }!!
 
         "gsettings set $settingsPath $settingsKey Adwaita".runCommand()
-        assertEquals("Adwaita", service.currentGtkTheme)
```

Comment:
I removed this line because the existing test case does not quite make sense to me:

we decrement the `countDownLatch` in the eventhandler's callback.

however, with this line here, we don't really wait for the latch to assert, making the test itself subject to race conditions.